src/components/Concept/ConceptPage.jsx

```diff
@@ -153,17 +158,13 @@ const ConceptPage = ({ conceptId, handleClose, inModal, language }) => {
     </Modal>
   );
 
-  const StyledVisualElementContainer = styled.div`
-    flex-grow: 1;
-  `;
```

Comment:
Denne blei vel lagt inn for å få svg-bilder til å vise alltid. Trengs ikkje det lenger?

Reply:
Bra du så dette. Jeg har nå laget en bedre løsning som både håndterer disse svg-ene og som midtstiller elementer som har full bredde.

Tilsvarende fiks på ed: https://github.com/NDLANO/editorial-frontend/pull/1202